neo/test/iotest/test_asciisignalio.py

```diff
@@ -13,6 +13,7 @@
 from neo.test.iotest.common_io_test import BaseTestIO
 from neo.core import AnalogSignal, Segment, Block
 
+import string
```

Comment:
I don't think this change is related to the NWBIO change you suggest. In that case can you remove it here?

Reply:
Yes. It is done. I removed it.